Gemfile

```diff
@@ -32,7 +32,16 @@ gem 'rspec-collection_matchers', '~> 1.1'
 gem 'rspec_junit_formatter', '>= 0.4.1'
 gem 'rspec_n', '~> 1.3' if RUBY_VERSION >= '2.3.0'
 gem 'ruby-prof', '~> 1.4' if RUBY_PLATFORM != 'java' && RUBY_VERSION >= '2.4.0'
-gem 'simplecov', '~> 0.17'
+if RUBY_VERSION >= '2.5.0'
+  # Merging branch coverage results does not work for old, unsupported rubies.
+  # We have a fix up for review, https://github.com/simplecov-ruby/simplecov/pull/972,
+  # but given it only affects unsupported version of Ruby, it might not get merged.
+  gem 'simplecov', git: 'https://github.com/marcotc/simplecov', ref: '3bb6b7ee58bf4b1954ca205f50dd44d6f41c57db'
```

Comment:
Since this may stick for a few years (it sounds like there'll be a manned mission to Mars before we drop 2.4), I suggest forking this to the datadog account, and pushing the branch there as well, just in case :)